packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -226,14 +232,31 @@ export function parseStylesVariables( styles, mappedValues, customValues ) {
 				] );
 			} );
 		}
+
+		if ( variable === 'var' ) {
+			stylesBase = stylesBase.replace( varRegex, ( _$1, $2 ) => {
+				if ( mappedValues && mappedValues?.color ) {
```

Comment:
Nitpick: We only need one condition when using optional chaining.
```suggestion
				if ( mappedValues?.color ) {
```

Reply:
Thanks for the suggestion! I applied the change 👍